pipelines/cpBias.yaml

```diff
@@ -25,6 +25,6 @@ tasks:
       calibrationType: 'bias'
       exposureScaling: "Unity"
 contracts:
-  - isr.doBias == False
```

Comment:
Contracts are "expressions written with Python syntax that should evaluate to a boolean" according to https://github.com/lsst/pipe_base/blob/main/doc/lsst.pipe.base/creating-a-pipeline.rst#verifying-configuration-contracts , so these should stay as `False`.